crates/hc_sandbox/src/run.rs

```diff
@@ -159,8 +150,35 @@ async fn start_holochain(
     Ok((holochain, lair))
 }
 
+async fn start_lair(passphrase: &[u8], lair_path: PathBuf) -> anyhow::Result<Child> {
+    use tokio::io::AsyncWriteExt;
+
+    tracing::info!("\n\n----\nstarting lair\n----\n\n");
+    let mut cmd = Command::new("lair-keystore");
+    cmd.arg("--lair-root")
+        .arg(lair_path)
+        .arg("server")
+        .arg("--piped")
+        .stdin(Stdio::piped())
+        .stdout(Stdio::piped())
+        .stderr(Stdio::inherit())
+        .kill_on_drop(true);
+
+    msg!("{:?}", cmd);
```

Comment:
Is this intentionally kept here?

Reply:
I didn't add this, I just extracted the function. I'm not sure if it's deliberate/useful. It does look like it was for debugging so I'll remove it